flake.nix

```diff
@@ -98,6 +98,7 @@
                 # Release flags
                 cmakeFlags = [
                   "-DCMAKE_BUILD_TYPE=Release"
+                  "-DCMAKE_CXX_FLAGS=-Werror=missing-field-initializers"
```

Comment:
So I think the scope is wider than just CI - we'd like anyone building without nix to hit this too, right? If so, probably throw this in here? https://github.com/bpftrace/bpftrace/blob/3e915ae21d32418ac56f4e6174f96918a5926d9e/CMakeLists.txt#L36-L46

Reply:
I know *nothing* about nix and only found the cmakeFlags while spelunking. :)
During testing I added -DCMAKE_CXX_FLAGS to verify, but I agree it makes sense to add the -Werror bit to all builds. Will do.